internal/scorecard/alpha/testpod.go

```diff
@@ -89,13 +90,13 @@ func getPodLog(client kubernetes.Interface, pod *v1.Pod) (logOutput []byte, err
 	return buf.Bytes(), err
 }
 
-func deletePods(client kubernetes.Interface, tests []Test) {
-	for _, test := range tests {
-		p := test.TestPod
-		err := client.CoreV1().Pods(p.Namespace).Delete(context.TODO(), p.Name, metav1.DeleteOptions{})
-		if err != nil {
-			log.Errorf("Error deleting pod %s %s\n", p.Name, err.Error())
-		}
-
+func deletePods(o Scorecard) {
```

Comment:
Probably better to make this a method on Scorecard, and have it return an error, which we can log at a higher spot in the stack.

```suggestion
func (s Scorecard) deletePods(ctx context.Context) error {
```

Reply:
done.